### PR TITLE
Fixed the bug with the label

### LINKED ISSUE #69: [Password reset] The “Email” input field is overlapped by description text.
**Description:** The “Email” input field is overlapped by description text on the password reset pop-up.

**Environment:** Windows 10 x64, Microsoft Edge 115.0.1901.203, Google Chrome 116.0.5845.96.

**Reproducible:** always.

**Build found:**

**Steps to reproduce:**
1. Click on the “Login” button on the header of the home page.
2. Click on the “Forgot password?” button on the “Password reset” pop-up.
3. Take a look at the “Email” input field.

**Actual result:**
The “Email” input field is overlapped by description text on the password reset pop-up.
Image

**Expected result:**
The “Email” input field is displayed aligned on the password reset pop-up.
Image

**User story and test case links**
User Story User password reset #13

### PR DESCRIPTION
**Pull request by holus-bolus**

### CODE REVIEW

**Review by iaremchuk — APPROVED** [review]
Sets the margin-top (mt) to '15px'.

**Review by StasRusnak1 — APPROVED** [review]

File: src/containers/guest-home-page/forgot-password/ForgotPassword.jsx [hunk 1]
```diff
@@ -94,7 +94,7 @@ const ForgotPassword = () => {
           onChange={handleInputChange('email')}
           required
           size='large'
-          sx={{ mb: '5px' }}
+          sx={{ mb: '5px', mt: '15px' }}
```
Comment (iva-stasia) [inline src/containers/guest-home-page/forgot-password/ForgotPassword.jsx]: It seems that according to the design, the margin-top should be about 30px.

**Review by iva-stasia — CHANGES_REQUESTED** [review]

**Review by iva-stasia — APPROVED** [review]

File: src/containers/guest-home-page/forgot-password/ForgotPassword.jsx (hunk 1 above)
Comment (holus-bolus) [reply src/containers/guest-home-page/forgot-password/ForgotPassword.jsx]: Yes, my bad, it's 25, not 15

Comment (vldkk) [reply src/containers/guest-home-page/forgot-password/ForgotPassword.jsx]: I see 32px
image

Comment (holus-bolus) [reply src/containers/guest-home-page/forgot-password/ForgotPassword.jsx]: image

Comment (holus-bolus) [reply src/containers/guest-home-page/forgot-password/ForgotPassword.jsx]: image


Here it is even 36px. How do we measure and what to add?

**Review by vldkk — APPROVED** [review]
Good job!

**Review by ashfox21 — APPROVED** [review]
Nicely done!

**Review by Wiret02 — APPROVED** [review]

**Review by online-marathon — APPROVED** [review]

**Pull request merged**